lua/neogit/status.lua

```diff
@@ -586,49 +596,25 @@ local function current_line_is_hunk()
   return h ~= nil
 end
 
-local function get_hunk_of_item_for_line(item, line)
-  if item.hunks == nil then
-    return nil
-  end
-
-  local hunk
-  local lines = {}
-  for _, h in ipairs(item.hunks) do
-    if h.first <= line and line <= h.last then
-      hunk = h
-      for i = hunk.diff_from, hunk.diff_to do
-        table.insert(lines, item.diff.lines[i])
-      end
-      break
-    end
-  end
-  return hunk, lines
-end
-
-local function get_current_hunk_of_item(item)
-  if item.hunks == nil then
-    return nil
-  end
-  return get_hunk_of_item_for_line(item, vim.fn.line("."))
-end
-
 local function toggle()
-  local section, item = get_current_section_item()
-  if section == nil then
+  local sel = M.get_selection()
```

Comment:
Do you mind using the full word "selection"? Makes it easier for me to read :)

Reply:
Yes absolutely, my fingers have typed `self` instead way to many times